test/factories/index.js

```diff
@@ -3,11 +3,13 @@ import factoryGirl from 'factory-girl'
 
 import userFactoryDefine from './user'
 import inviteCodeFactoryDefine from './inviteCode'
+import userAvatarFactoryDefine from './userAvatar'
 
 const factory = factoryGirl.promisify(bluebird)
 factory.setAdapter(new factory.ObjectAdapter())
```

Comment:
We need to change this to a thinky adapter in order for `factory.create(...)` to work properly.

Similar to https://github.com/LearnersGuild/echo/blob/45d1b79ec7666361d86703c7a3ff4a2c77c340d8/src/test/factories/RethinkDBAdapter.js.